src/main/java/org/mastodon/mamut/feature/branch/BranchNSpotsFeature.java

```diff
@@ -46,13 +41,141 @@
+ * be equivalent to {@link BranchDisplacementDurationFeature} (duration).
+ * However, in situations where there are spots missing within a branch spot, this
+ * may well be different.
+ *
+ * <h1>Model-Graph (i.e. Graph of Spots)</h1>
+ *
+ * <pre>
+ *                                                Spot( 0, X=1,00, Y=2,00, Z=3,00, tp=0 )
+ *                                                                   │
+ *                                                                   │
+ *                                                Spot( 1, X=2,00, Y=4,00, Z=6,00, tp=1 )
+ *                                                                   │
+ *                                                                   │
+ *                                                Spot( 2, X=3,00, Y=6,00, Z=9,00, tp=2 )
+ *                       ┌───────────────────────────────────────────┴──────────────────────┐
+ *                       │                                                                  │
+ *  Spot( 11, X=12,00, Y=24,00, Z=36,00, tp=3 )                         Spot( 3, X=4,00, Y=8,00, Z=12,00, tp=3 )
+ *                       │                                                                  │
+ *                       │                                                                  │
+ *  Spot( 12, X=13,00, Y=26,00, Z=39,00, tp=4 )                         Spot( 4, X=5,00, Y=10,00, Z=15,00, tp=4 )
+ *                       │                                            ┌─────────────────────┴─────────────────────┐
+ *                       │                                            │                                           │
+ *  Spot( 13, X=14,00, Y=28,00, Z=42,00, tp=5 )   Spot( 8, X=9,00, Y=18,00, Z=27,00, tp=5 )   Spot( 5, X=6,00, Y=12,00, Z=18,00, tp=5 )
+ *                                                                    │                                           │
+ *                                                                    │                                           │
+ *                                               Spot( 9, X=10,00, Y=20,00, Z=30,00, tp=6 )   Spot( 6, X=7,00, Y=14,00, Z=21,00, tp=6 )
+ *                                                                    │                                           │
+ *                                                                    │                                           │
+ *                                               Spot( 10, X=11,00, Y=22,00, Z=33,00, tp=7 )  Spot( 7, X=8,00, Y=16,00, Z=24,00, tp=7 )
+ * </pre>
+ *
+ * <h1>Branch-Graph (i.e. Graph of BranchSpots)</h1>
+ *
+ * <pre>
+ *                        branchSpotA
+ * 	       ┌──────────────┴─────────────────┐
+ * 	       │                                │
+ * 	   branchspotB                      branchSpotC
+ * 	                                 ┌──────┴───────┐
+ * 	                                 │              │
+ *                                 branchspotD branchSpotE
```

Comment:
Type and some spaces missing. The same is true for the other javadoc comments.

Reply:
Changed it.